docs/content/querying/lookups.md

```diff
@@ -3,42 +3,82 @@ layout: doc_page
+Namespaced lookups can be specified as part of the runtime properties file. The property is a list of the namespaces 
+described as per the sections on this page. For example:
 
  ```json
- druid.query.extraction.namespace.lookups=\
-   [{ "type":"uri", "namespace":"some_uri_lookup","uri": "file:/tmp/prefix/",\
-   "namespaceParseSpec":\
-     {"format":"csv","columns":["key","value"]},\
-   "pollPeriod":"PT5M"},\
-   { "type":"jdbc", "namespace":"some_jdbc_lookup",\
-   "connectorConfig":{"createTables":true,"connectURI":"jdbc:mysql://localhost:3306/druid","user":"druid","password":"diurd"},\
-   "table": "lookupTable", "keyColumn": "mykeyColumn", "valueColumn": "MyValueColumn", "tsColumn": "timeColumn"}]
+ druid.query.extraction.namespace.lookups=
+   [
+     {
+       "type": "uri",
+       "namespace": "some_uri_lookup",
+       "uri": "file:/tmp/prefix/",
+       "namespaceParseSpec": {
+         "format": "csv",
+         "columns": [
+           "key",
+           "value"
+         ]
+       },
+       "pollPeriod": "PT5M"
+     },
+     {
+       "type": "jdbc",
+       "namespace": "some_jdbc_lookup",
+       "connectorConfig": {
+         "createTables": true,
+         "connectURI": "jdbc:mysql:\/\/localhost:3306\/druid",
+         "user": "druid",
+         "password": "diurd"
+       },
+       "table": "lookupTable",
+       "keyColumn": "mykeyColumn",
+       "valueColumn": "MyValueColumn",
+       "tsColumn": "timeColumn"
+     }
+   ]
  ```
 
-Proper funcitonality of Namespaced lookups requires the following extension to be loaded on the broker, peon, and historical nodes:
+Proper functionality of Namespaced lookups requires the following extension to be loaded on the broker, peon, and historical nodes:
 `io.druid.extensions:druid-namespace-lookup`
 
 ## Cache Settings
-The following are settings used by the nodes which service queries when setting namespaces (broker, peon, historical)
+
+Lookups are cached locally on historical nodes. The following are settings used by the nodes which service queries when 
+setting namespaces (broker, peon, historical)
 
 |Property|Description|Default|
 |--------|-----------|-------|
 |`druid.query.extraction.namespace.cache.type`|Specifies the type of caching to be used by the namespaces. May be one of [`offHeap`, `onHeap`]. `offHeap` uses a temporary file for off-heap storage of the namespace (memory mapped files). `onHeap` stores all cache on the heap in standard java map types.|`onHeap`|
 
-The cache is populated in different ways depending on the settings below. In general, most namespaces employ a `pollPeriod` at the end of which time they poll the remote resource of interest for updates. The notable exception being the kafka namespace lookup as defined below.
+The cache is populated in different ways depending on the settings below. In general, most namespaces employ 
+a `pollPeriod` at the end of which time they poll the remote resource of interest for updates. The notable exception 
```

Comment:
I know this was already here, but could you fix this non-sentence?  You could say "The notable exception is the Kafka namespace lookup ..." if it's the only exception, or "A notable exception" if there are others.